.github/workflows/coverity.yml

```diff
@@ -0,0 +1,32 @@
+# Creates and uploads a Coverity build on a schedule
+name: Coverity Scan
+on:
+  #schedule:
+  #  # Run twice daily, since below 500k LOC can have 21 builds per week, per https://scan.coverity.com/faq#frequency
+  #  - cron: '0 */12 * * *'
```

Comment:
i suppose i need to uncomment this?
p.s. i think once a day is enough (our change rate is not high enough to justify more)

Reply:
Thanks, this is what I get for committing on a Friday afternoon 😄 .